dcrctl/go.mod

```diff
@@ -2,7 +2,7 @@ module decred.org/release/v2/dcrctl
 
 go 1.17
```

Comment:
Well, I approved it already, but now I'm wondering if this needs to be go1.24 for the `tool` directive (and same with the others).  It works fine with a `go.work` which requires go1.24, but without it?

Reply:
nope, not needed